pkg/frontend/adminactions/delete_managedresource.go

```diff
@@ -52,6 +53,14 @@ func (a *azureActions) ResourceDeleteAndWait(ctx context.Context, resourceID str
 		return a.deleteFrontendIPConfiguration(ctx, resourceID)
 	}
 
+	re = regexp.MustCompile(healthProbePattern)
+	// HealthProbes cannot be deleted with DeleteByIDAndWait either.
+	matches := re.FindStringSubmatch(resourceID)
```

Comment:
Can we give `matches` a more descriptive name?

Reply:
I ended up removing matches all together